.github/workflows/ci.yaml

```diff
@@ -0,0 +1,41 @@
+name: Continuous Integration
+
+on:
```

Comment:
When will all this run? On pushes to master (if someone pushed directly) and on PRs to the `v1-mvp` branch